### PR TITLE
enable login

### PR DESCRIPTION
**Pull request by suhaybjirde2**
check the authentication of the users

### CODE REVIEW

**Comment by suhaybjirde2** [comment]
check out this new feature

File: login.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+login alowed
```
Comment (suhaybjirde) [inline login.txt:1]: pls add some python code

**Review by suhaybjirde — CHANGES_REQUESTED** [review]
make those changes

File: file.txt [hunk 2]
```diff
@@ -1 +1,2 @@
 hello 
+print('hello world')
```
Comment (suhaybjirde) [inline file.txt:2]: well done tha's what i'm talking about

**Review by suhaybjirde — APPROVED** [review]
good job

**Pull request merged**